### PR TITLE
Introduce github actions for CI

### PR DESCRIPTION
**Pull request by turbaszek**
This is a followup PR for #44

### CODE REVIEW

**Comment by turbaszek** [comment]
@sharanf can we please ask infra team to turn on Github Actions for Apache Kibble project? It cannot be configured via `.asf.yaml` file.

The CI seems to be working on my fork:
https://github.com/PolideaInternal/kibble/runs/1235577492

**Comment by sharanf** [comment]
@turbaszek I have requested a review from @Humbedooh who is part of the ASF Infra team so if there is anything missing I am sure he will let us know :-)

**Review by michalslowikowski00 — APPROVED** [review]
Great! :)

**Comment by turbaszek** [comment]
@Humbedooh any update on the possibility of turning on Github Actions for the Apache Kibble project?

**Comment by Humbedooh** [comment]
GitHub actions should already be enabled for Kibble

**Comment by turbaszek** [comment]
> GitHub actions should already be enabled for Kibble

Hm, interesting. @Humbedooh do we "Allow all actions" or only "Allow local actions only"? The yaml is ok as the CI is running on my fork:
https://github.com/PolideaInternal/kibble/actions

**Comment by Humbedooh** [comment]
Its allow all

**Comment by turbaszek** [comment]
@TobKed you are GA expert, any ideas why GA doesn't want to start? 😄

**Comment by TobKed** [comment]
> @TobKed you are GA expert, any ideas why GA doesn't want to start? 😄

I am not 100% sure but I think it will not run before merging. @Humbedooh is it possible that "Allow local actions only" is overridden by organization settings which may be "Allow all actions"?

**Comment by Humbedooh** [comment]
Both org and local setting is "allow all", so I dunno. But we'll see if it'll run once we merge, I guess :)

**Comment by turbaszek** [comment]
> Both org and local setting is "allow all", so I dunno. But we'll see if it'll run once we merge, I guess :)

Yup, here's reply from Github support:
> I'm showing that the forked repository has GitHub Actions workflow files, but I'm not seeing any on the main repository.
>
> You will need to merge your change that adds the ci.yml file to the main repository before it will be able to start running the action when you perform pull_request.

**Comment by turbaszek** [comment]
@Humbedooh so it seems we need to merge it to see CI up and running 🤷‍♂️ This will require a follow-up PR that will make CI green - I can make it asap after merging this

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -17,3 +17,29 @@ We also have:
 
 This project requires Python in higher version than 3.4.
 More information will come soon!
+
+## Code Quality
+
+Apache Kibble project is using [pre-commits](https://pre-commit.com) to ensure the quality of the code.
+We encourage you to use pre-commits, but it's not a required to contribute. Every change is checked
```
Comment (Humbedooh) [inline CONTRIBUTING.md]: Should be either `it's not required in order to contribute` or `it's not a requirement in order to contribute`.

**Comment by turbaszek** [comment]
Hi all, is there anything that should be done to get this PR merged? @Humbedooh @sharanf

**Comment by sharanf** [comment]
Hi @turbaszek as @Humbedooh  has already given feedback on this I am happy to merge it.

**Pull request merged**